appveyor.yml

```diff
@@ -0,0 +1,122 @@
+# Copyright 2016, 2017 Peter Dimov
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at http://boost.org/LICENSE_1_0.txt)
+
+# When copying this to a new library, be sure to update the name of the library
+# in two places (once each at the top of install: and test_script:)
+
+version: 1.0.{build}-{branch}
+
+shallow_clone: true
+
+branches:
+  only:
+    - develop
+    - master
+
+matrix:
+  allow_failures:
+    - MAYFAIL: true
+
+environment:
+  global:
+    # see: http://www.boost.org/build/doc/html/bbv2/overview/invocation.html#bbv2.overview.invocation.properties
+    # to use the default for a given environment, comment it out; recommend you build debug and release however..
+    # on Windows it is important to exercise all the possibilities, especially shared vs static
+    # B2_ADDRESS_MODEL: address-model=64,32
+    # B2_LINK: link=shared,static
+    B2_THREADING: threading=multi,single
+    B2_VARIANT: variant=release,debug
+
+  matrix:
+  # test older compilers
+    - TOOLSET: msvc-9.0
+      COMMENT: Visual Studio 2008
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    - TOOLSET: msvc-10.0
+      COMMENT: Visual Studio 2010
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    - TOOLSET: msvc-11.0
+      COMMENT: Visual Studio 2012
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    - TOOLSET: msvc-12.0
+      COMMENT: Visual Studio 2013
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    - ADDPATH: C:\cygwin\bin;
+      TOOLSET: gcc
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    - ADDPATH: C:\mingw\bin;
+      TOOLSET: gcc
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    - ADDPATH: C:\mingw-w64\x86_64-6.3.0-posix-seh-rt_v5-rev1\mingw64\bin;
+      TOOLSET: gcc
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+    - TOOLSET: msvc-14.0
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+  # test using windows headers
+    - COMMENT: Use Windows.H
+      TOOLSET: msvc-14.1
+      DEFINES: define="BOOST_USE_WINDOWS_H"
```

Comment:
You should not define this. The library tests are compiled multiple times in different configurations, one of them being with `BOOST_USE_WINDOWS_H` defined. By defining it in the command line you effectively disable tests that are supposed to be compiled without it.

Reply:
Okay great, that removes a build job - I'm all for that.